lib/file-view.coffee

```diff
@@ -19,7 +19,11 @@ class FileView extends HTMLElement
     @fileName.dataset.name = @file.name
     @fileName.dataset.path = @file.path
 
-    @fileName.classList.add(FileIcons.getService().iconClassForPath(@file.path))
+    iconClass = FileIcons.getService().iconClassForPath(@file.path, this)
+    if iconClass
+      unless Array.isArray iconClass
```

Comment:
> I think it's safe to assume many authors (particularly those used to working with jQuery) might want to pass multiple classes as a space-separated string.

I wonder if we could just deprecate the string option altogether, and just call this method `iconClassesForPath`, which always returns an array...

Anyways, I think it's okay to be less strict on this method and accept both space-separated strings as well as arrays, let's keep this as it is.